requirements.txt

```diff
@@ -10,3 +10,8 @@ numpy==1.26.4
 rawpy==0.21.0
 pillow-heif==0.16.0
 chardet==5.2.0
+pydub==0.25.1
+mutagen==1.47.0
+numpy==1.26.4
+ffmpeg-python==0.2.0
```

Comment:
I have seen this in another PR, but ffmpeg needs to be installed. That should be mentioned or written how you would do that.

Reply:
I'm addressing this in #390 👍